src/Microsoft.AspNetCore.Hosting/WebHostBuilderExtensions.cs

```diff
@@ -86,12 +86,12 @@ public static IWebHostBuilder UseStartup(this IWebHostBuilder hostBuilder, Type
         /// <param name="hostBuilder">The <see cref="IWebHostBuilder"/> to configure.</param>
         /// <param name="configure">A callback used to configure the <see cref="ServiceProviderOptions"/> for the default <see cref="IServiceProvider"/>.</param>
         /// <returns>The <see cref="IWebHostBuilder"/>.</returns>
-        public static IWebHostBuilder UseDefaultServiceProvider(this IWebHostBuilder hostBuilder, Action<ServiceProviderOptions> configure)
```

Comment:
Was added in https://github.com/aspnet/Hosting/commit/d57d729d13c73750027944dda25082693d6b6d7f it's new in 2.0